model/config.go

```diff
@@ -2348,8 +2364,9 @@ func ConfigFromJson(data io.Reader) *Config {
 	return o
 }
 
+// isNew guesses if the current config file is a new one or we are making an upgrade based on SiteURL being different from default or nil
 func (o *Config) isNew() bool {
-	return *(o.ServiceSettings.SiteURL) == SERVICE_SETTINGS_DEFAULT_SITE_URL
+	return o.ServiceSettings.SiteURL == nil || *(o.ServiceSettings.SiteURL) == SERVICE_SETTINGS_DEFAULT_SITE_URL
```

Comment:
There are /some/ installations -- unfortunately -- where `SiteURL` is non-nil and yet left at the default. Wondering if we can just check for the non-nil setting instead?